celery/backends/dynamodb.py

```diff
@@ -199,11 +199,11 @@ def _get_or_create_table(self):
             error_code = e.response['Error'].get('Code', 'Unknown')
 
             # If table exists, do not fail, just return the description.
-            if error_code == 'ResourceInUseException':
+            try:
                 return self._client.describe_table(
                     TableName=self.table_name
                 )
-            else:
+            except:
```

Comment:
I'm not a fan of bare excepts.
Is there a common base class for these type of exceptions?